app/api/Controller.py

```diff
@@ -128,4 +127,13 @@ def editEvent(self, email, name, newevent):
         if resp.get('success'):
             return {'success':True, 'payload':resp.get('message')}
         return {'success':False, 'message':resp.get('message')}
-                      
\ No newline at end of file
+    def acceptRsvp(self, userId, eventId, clientEmail):
```

Comment:
class methods or functions should be written in small caps with each word being separated by underscores e.g. accept_rsvp instead of acceptRsvp.